admin/tools/provision/cloudmanager.py

```diff
@@ -0,0 +1,311 @@
+    # parser = lsst.qserv.admin.logger.add_logfile_opt(parser)
+    group = parser.add_argument_group('Cloud configuration options',
+                                       'Options defining parameters to access remote cloud-platform')
+
+    group.add_argument('-f', '--config', dest='configFile',
+                        required=True, metavar='PATH',
+                        help='Add cloud config file which contains instance characteristics')
+
+    return parser
+
+def config_logger(loggerName, verbose, verboseAll):
+    """
+    Configure the logger
+    """
+    verbosity = len(verbose)
+    levels = {0: logging.WARNING, 1: logging.INFO, 2: logging.DEBUG}
+    if not verboseAll:
+        # suppress INFO/DEBUG regular messages from other loggers
+        # Disable requests and urllib3 package logger and warnings
+        logging.getLogger("requests").setLevel(logging.ERROR)
+        logging.getLogger("urllib3").setLevel(logging.ERROR)
+
+    logging.basicConfig(format='%(asctime)s %(levelname)-8s %(name)-15s'
+                               ' %(message)s',
+                        level=levels.get(verbosity, logging.DEBUG))
+
+    warnings.filterwarnings("ignore")
+
+
+class CloudManager(object):
+    """Application class for common definitions for creation of snapshot and provision qserv"""
+
+    def __init__(self, config_file_name, used_image_key=BASE_IMAGE_KEY, add_ssh_key=False):
+        """
+        Constructor parse all arguments
+
+        @param add_ssh_key Add ssh key only while launching instances in provision qserv.
+        @param go_for_snapshot find new_image until it's created
+        """
+
+        logging.debug("Use configuration file: {}".format(config_file_name))
+
+        config = ConfigParser.ConfigParser({'net-id': None,
+                                            'ssh_security_group': None})
+
+        try:
+            config_file = open(config_file_name, 'r')
+            try:
+                config.readfp(config_file)
+            finally:
+                config_file.close()
+        except IOError as exc:
+            logging.fatal('Unable to read configuration file: {}'.format(exc))
+            sys.exit(1)
+
+        self._creds = get_nova_creds()
+        logging.debug("Openstack user: {}".format(self._creds['username']))
+        self._safe_username = self._creds['username'].replace('.', '')
+        self.nova = client.Client(**self._creds)
+
+        base_image_name = config.get('openstack', used_image_key)
+        self.snapshot_name = config.get('openstack', 'snapshot_name')
+        self.image = self.nova.images.find(name=base_image_name)
+
+        flavor_name = config.get('openstack', 'flavor_name')
+        self.flavor = self.nova.flavors.find(name=flavor_name)
+
+        self.network_name = config.get('openstack', 'network_name')
+        if config.get('openstack', 'net-id'):
+            unicode_net_id = unicode(config.get('openstack', 'net-id'))
```

Comment:
Can't we just write admin scripts like these in python 3 and forget about python 2?